data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -222,7 +222,7 @@
                                     [message]
                                         speaker=narrator
                                         image="wesnoth-icon.png"
-                                        message= _ "You struggle to lift and don the heavy plate. Once worn, however, it is amazingly comfortable. You have increased resistance to all physical damage!"
+                                        message= _ "$unit.name| struggles to lift and don the heavy plate. Once worn, however, it is amazingly comfortable. $unit.name| feels an increased resistance to all physical damage!"
```

Comment:
I thought about it and went with "least change". I would prefer the unit to speak as with the sword, but didn't want to make a large style change.

I will re-word to be more like the flaming sword.  "He" and "She" would work better in the latter phrase.

I can make a unit with no name and, yes, expect it to have a problem. But I don't think it's possible for the player, here, to have one without debug mode.